webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -6445,6 +6450,11 @@ const firstVisibleMsg = {
                     data: w.JSON.stringify(messagePxy),
                     contentType: 'application/json',
                     success: function (data) {
+                        const {groupId} = messagePxy;
+
```

Comment:
are you expecting only one group for the customer and that too always opened in that tab?